Source/WebKit/PlatformGTK.cmake

```diff
@@ -369,7 +369,7 @@ if (ENABLE_MEDIA_STREAM)
     )
 endif ()
 
-if (${GI_VERSION} VERSION_GREATER_EQUAL 1.79.2)
+if ("${GI_VERSION}" VERSION_GREATER_EQUAL 1.79.2)
```

Comment:
Maybe it would be less confusing to just check `if (ENABLE_INTROSPECTION AND GI_VERSION VERSION_GREATER_EQUAL 1.79.2)`

Unless I'm mistaken, we shouldn't even need to defereference `${}` the `GI_VERSION` here, right? I wonder why I did that. Maybe it really is needed?